model-optimizer/mo/utils/unittest/graph.py

```diff
@@ -330,7 +330,7 @@ def get_name_and_port(tensor_name):
         return node_name, 0
 
 
-def connect(first_tensor_name, second_tensor_name, skip_data=False):
+def connect(first_tensor_name, second_tensor_name, skip_data=False, on_front=False):
```

Comment:
Add a comment describing a new parameter.
Can we rename it to `front_phase`?

Reply:
Renamed to `front_phase` and also added alias function `connect_front` . Please review.